src/features/flow/connect/Select.tsx

```diff
@@ -128,6 +152,16 @@ export default () => {
                     items={items}
                     onSelect={item => setSelectedSerialNumber(item.id)}
                 />
+                <br />
```

Comment:
Please do not make empty lines by `<br />`, but use tailwind padding. So create additional divs and use `tw-flex tw-flex-col tw-gap-X` where X is a padding.